integreat_cms/static/src/js/bulk-actions.ts

```diff
@@ -28,23 +28,27 @@
  *
  */
 
+var bulkActionButton: HTMLButtonElement;
+var bulkAction: HTMLSelectElement;
```

Comment:
Is it really necessary to define these as global variables and set them on window load?
I think it would be better to just do the `document.getElementById("bulk-action")` every time these variables are used (and as far as I can see, it's only one location anyway?)